src/app/pages/Designathons/Designathon24/Designathon24.module.scss

```diff
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap");
+
```

Comment:
Chore: could we also remove this Google Fonts import for now as we haven't decided how to proceed with ducimain vs DM Sans?

Issue (for future reference): using `@import` delays loading fonts since the stylesheet must be fetched first. Consider using `<link>` in the `<head>` of the document and also including the preconnect domains suggested by Google Fonts.